packages/jest-runtime/src/index.ts

```diff
@@ -808,9 +808,6 @@ class Runtime {
 
     if (transformedFile.sourceMapPath) {
       this._sourceMapRegistry[filename] = transformedFile.sourceMapPath;
-      if (transformedFile.mapCoverage) {
-        this._needsCoverageMapped.add(filename);
```

Comment:
can we delete the `_needsCoverageMapped` property?

Reply:
I pulled on this thread a bit more - we can remove the property and there's an entire `getSourceMapInfo` function which collapses down into returning an empty object.

There was a call site in `runTest` which used this to attach a `sourceMaps` property to `TestResult`, leading to some more apparently defunct code in `coverage_reporter`.

Deleted a few more things here and added a couple more TODOs for deletion in Jest 26.